### ISSUE TITLE
Showing "lighthouse is warming up" forever

### ISSUE THREAD

**Issue by torfmaster** [GoogleChrome/lighthouse#9484]
### Maintainer's Note:
tl;dr - This happens when Chromium fails to fetch the Lighthouse code from the web. Check your network connectivity, ensure there are no proxies that would prevent you from downloading code from a Google server, and try again. 

On Brave, you'll have to enable the `Remote Debugging` option under `brave://settings/privacy` (thx @bsclifton!)

---------
### Original Issue:

When running lighthouse on Chromium "75.0.3770.90" to audit www.google.com it is showing the message "lightouse is warming up" forever. Since 2019-07-30. It used to worked in the very exact setup before. 
Probably, infrastructure to run lighthouse is unavailable.

**Comment by patrickhulce** [comment]
What is your operating system @torfmaster?

Also try updating your Chrome version as 75.0.3770.90 is running behind latest stable.

**Comment by connorjclark** [comment]
This happens when the DT uploader fails. I'm not sure how to check if Lighthouse for a specific version of Chromium uploaded successfully. But it seems the latest versions are good. Please update Chrome and let us know if it is still broken.

**Comment by patrickhulce** [comment]
IIRC @connorjclark I believe this also happens in certain distributions of Chromium in unusual linux environments, sorta like the #7246 failure but when they disabled remote modules.

**Comment by torfmaster** [comment]
The operating system is ubuntu 18.04. @patrickhulce I doubt this is caused by my setup as it used to work and stopped working out of the sudden (without having installed any updated of chromium or lighthouse).

**Comment by patrickhulce** [comment]
In this case yes it is almost definitely the Chrome version. I'm just saying that in general, this issue also has potential interaction with the OS-specific distribution of Chromium. 

Did updating to latest stable fix the issue for you @torfmaster?

**Comment by aladdin-add** [comment]
same issue!

browser: 0.68.132 Chromium: 76.0.3809.132
os: macos 10.14.5 (18F132)

please let me know if some more info is needed.

**Comment by brendankenny** [comment]
Closing as there hasn't been movement in some time. Please comment and we can reopen if you're still seeing this issue.

**Comment by ghonchesefidi** [comment]
Hi
I still have this problem:
Version 77.0.3865.90 (Official Build) (64-bit)
Linux Ubuntu 18.04.03

**Comment by gjunming** [comment]
@brendankenny , I have the same problem on macOS Mojave (version 10.14.6) with Chrome Version 77.0.3865.120 (Official Build) (64-bit)

**Comment by Morgan-Marshall** [comment]
Just seeing this issue after updating to Chrome Version 78.0.3904.70 , Windows 10

**Comment by jj-wd** [comment]
Have that same issue. Started using Canary for Lighthouse Audits now as a "workaround".

**Comment by nmrony** [comment]
I am also facing same issue .
Version 0.70.121 Chromium: 78.0.3904.70 (Official Build) (64-bit)
MacOS: 10.15.1

**Comment by falldowngoboone** [comment]
I am having this issue as well. Additionally, I cannot get anything to show up in the Accessibility pane in either Chrome or Chrome Canary.

MacOS 10.14.6
Chrome 78.0.3904.87

**Comment by connorjclark** [comment]
Thanks for reporting! I have a theory, but could use your help to verify. I presume you have just updated / downloaded the newest Stable Chrome? Could you try (a few times) completely killing Chrome, restarting, and running Lighthouse via DevTools?

**Comment by jj-wd** [comment]
I wanted to reproduce the issue before trying to kill the process but it was working again immediately. I've meanwhile rebooted and cleared the cache a dozen times though.

**Comment by gab4mad** [comment]
In my case it was very simple, but you have to come to this solution...

I had this issue on Windows10 pro with the actual chrome version.
Just startd chrom as admin an the whole thing worked....

I have an issue when working with debian, I guess admin right are needed too, but htis is not verified, but I will try this tomorrow....

**Comment by antwake** [comment]
Doesn't work for me as well
Mojave 10.14.6 / Chromium: 78.0.3904.97 

stuck on "lighthouse is warming up"

**Comment by jj-wd** [comment]
Meanwhile the error reappered on my end. I could test the restarting / killing theory and it didn't start working even after a couple of times with and without clearing the cache.
I can confirm that it works as admin. I don't have another user to test with to check whether it's just admin or the user context overall.
Canary still works, even with the same sites.

**Comment by Mr-Zer0** [comment]
I am also experiencing the same issue. The lighthouse never stops warming up.
It happens on both windows 10 and Mac OS Catalina
Both chrome and brave versions are 78.0.3904.108 (Official Build).

**Comment by condensed-io** [comment]
same for me using brave browser: Version 1.0.0 Chromium: 78.0.3904.97 (Official Build) (64-bit)

**Comment by datanotion** [comment]
Just adding to the version pool:

Brave - Version 1.0.1 Chromium: 78.0.3904.108 (Official Build) (64-bit)
Catalina 10.15.1 (19B88)

**Comment by johnweland** [comment]
Brave - Version 1.0.1 Chromium: 78.0.3904.108 (Official Build) (64-bit) - fails
Chrome - Version 78.0.3904.108 (Official Build) (64-bit) - works

OS - Pop!_OS 19.10

Chromium Version 78.0.3904.97 (Developer Build) built on Debian buster/sid, running on Debian buster/sid (64-bit) - fails as well but with this error

Protocol error (Tracing.start): 'Tracing.start' wasn't found
```
Channel: DevTools
Initial URL: http://127.0.0.1:5500/index.html
Chrome Version: 78.0.3904.97
Stack Trace: Error: Protocol error (Tracing.start): 'Tracing.start' wasn't found
    at Function.fromProtocolMessage (devtools://devtools/remote/serve_file/@021b9028c246d820be17a10e5b393ee90f41375e/audits_worker/audits_worker_module.js:1603:121)
    at eval (devtools://devtools/remote/serve_file/@021b9028c246d820be17a10e5b393ee90f41375e/audits_worker/audits_worker_module.js:1209:253)
```

**Comment by patrickhulce** [comment]
@johnweland, Debian's build of Chromium intentionally removes Lighthouse + tracing support see #7246

**Comment by jd1378** [comment]
Having this isssue on Chrome Version 78.0.3904.108 (Official Build) (64-bit) on Windows 10 Home (Build 18362 or May 2019 Update)

**Comment by johnweland** [comment]
> @johnweland, Debian's build of Chromium intentionally removes Lighthouse + tracing support see #7246

I see. well it seems to be doing the same thing on Windows 1909 as well. On Brave and Chrome,  I don't have chromium on this system.

**Comment by Bert0324** [comment]
Do have the same problem in 

 - Chrome:78.0.3904.10
 - MacOS: 10.15.2 Beta

**Comment by vkaracic** [comment]
I have the same issue on Ubuntu 18.04. on Brave browser:
`Version 78.0.3904.108 (Official Build) (64-bit)`

Tried with the Chrome browser (which is the same version as above) and it worked. Guess Brave is no longer supported (properly). Should we move the conversation over to the brave browser repo? There is already one report https://github.com/brave/brave-browser/issues/7220

**Comment by journeymanavi** [comment]
I had the same problem today:
Chrome: Version 78.0.3904.108 (Official Build) (64-bit)
MacOS: 10.14.5 (18F132)

After struggling for some time I installed lighthouse npm package and [ran it from the command line](https://developers.google.com/web/tools/lighthouse#cli) and it worked! And, oddly, after that it also started working from within DevTools.

Just though of sharing in case this works for other too.

**Comment by bsclifton** [comment]
Hi folks - Brave developer here 😄 

For Brave, this is a known issue that was captured with https://github.com/brave/brave-browser/issues/3199. We had disabled remote debugging because it was making outbound calls

I recently resolved this in Brave by adding a new `Enable Remote Debugging` toggle in brave://settings/privacy. The fix is only in version 1.3 and greater (current stable version is 1.0.1). Fix is here for those curious:
https://github.com/brave/brave-core/pull/4044

**Comment by shepherd1530** [comment]
same problem here

- Chrome 78.0.3904.108 

- windows 10 pro

**Comment by exterkamp** [comment]
from #10103:  OS/windows 7 & Chrome Version/79.0.3945.79

**Comment by connorjclark** [comment]
Has anyone gotten this from running the latest Canary (81.x) or Dev (80.x) channels? This may have been resolved.

**Comment by mfkvfn** [comment]
Same problem here.
Windows 10 64 bit.
 Chrome version: 80.0.3970.5


> Has anyone gotten this from running the latest Canary (81.x) or Dev (80.x) channels? This may have been resolved.

**Comment by kazpr98** [comment]
Same problem here on the newest update. 

Chrome Version 81.0.4016.0 (Official Build) canary (64-bit).

OS: Mojave 10.14.6

**Comment by Lespinel82** [comment]
Same problem with:

Chrome Version 79.0.3945.117 (Official Build) (64-bit)
OS: Windows 10

**Comment by sadvakasov** [comment]
I've updated my Chrome on Windows 10 from 79.0.3945.117 to the latest 79.0.3945.130 and the problem is still appears

**Comment by genehack** [comment]
I am also seeing this on Win10 and Catalina with 79.0.3945.130. 

FWIW, I was able to use the Node CLI to successfully run a report (see https://github.com/GoogleChrome/lighthouse#using-the-node-cli for instructions), so that may be a workaround for folks that are stuck.

**Comment by ssayanm** [comment]
> Hi folks - Brave developer here 
> 
> For Brave, this is a known issue that was captured with [brave/brave-browser#3199](https://github.com/brave/brave-browser/issues/3199). We had disabled remote debugging because it was making outbound calls
> 
> I recently resolved this in Brave by adding a new `Enable Remote Debugging` toggle in brave://settings/privacy. Once you turn this on, it'll work again The fix is only in version 1.2 and greater (current stable version is 1.1.23; 1.2 is expected to hit Stable on Jan 6).
> 
> Fix is here for those curious:
> [brave/brave-core#4044](https://github.com/brave/brave-core/pull/4044)

Worked for me on debain 10 with brave

**Comment by holysoles** [comment]
Happening for me as well on:

Chrome Version 79.0.3945.130 (Official Build) (64-bit)
Windows Version: 10.0.18362 N/A Build 18362

**Comment by davinderpalrehal** [comment]
Happening to me as well, funnily its working fine in Edge

Chrome version: Version 81.0.4033.2 (Official Build) dev (64-bit)
Mac OS: Mojave Version 10.14.6

Problem occurs both in incognito mode and normal mode

Same issue in Opera and Brave.

**Comment by PortStone** [comment]
For me as well.

Google Chrome: Version 79.0.3945.130 (Official Build) (64-bit)
OS: Manjaro 18.1.5 Juhraya

**Comment by midzer** [comment]
Jup, can confirm @davinderpalrehal 

Chrome Version 81.0.4033.2 (Official Build) dev (64-bit)
on Debian

**Comment by davinderpalrehal** [comment]
@midzer not yet, updated to Chrome Version 81.0.4040.5 (Official Build) dev (64-bit) on Mac OS: Mojave v 10.14.6

Lighthouse is only working with Edge but not working on Chrome, Brave and Opera

**Comment by cjhahaha** [comment]
Mine got stuck at 'lighthouse is warming up' as well.

Version 80.0.3987.87 (Official Build) (64-bit)
OS: macOS Catalina Version 10.15.2

**Comment by KissBalazs** [comment]
Same issue, on an older machine.

Version: 80.0.3987.87 32 bit
OS: Windows 7

**Comment by xPaw** [comment]
This broke for me on 79, and is still broken on 80.0.3987.87.

OS: Windows 10 OS Version 1909

**Comment by cjhahaha** [comment]
> Mine got stuck at 'lighthouse is warming up' as well.
> 
> Version 80.0.3987.87 (Official Build) (64-bit)
> OS: macOS Catalina Version 10.15.2

I changed a network and then it worked.

**Comment by Tschoepel** [comment]
Chromium Version 80.0.3987.87 (Official Build) Arch Linux (64-bit)
OS: Manjaro 18.1.5
Also not working...

**Comment by xPaw** [comment]
I have adblock running on my local network, and `chrome-devtools-frontend.appspot.com` is blocked in AdAway list, whitelisting that has fixed lighthouse for me.

Lighthouse ought to have a proper error message when it's unable to fetch from there.

**Comment by PortStone** [comment]
> I have adblock running on my local network, and `chrome-devtools-frontend.appspot.com` is blocked in AdAway list, whitelisting that has fixed lighthouse for me.
> 
> Lighthouse ought to have a proper error message when it's unable to fetch from there.

That fixed it for me.

**Comment by davinderpalrehal** [comment]
Disabled the ad blocker but still stuck on the `Lighthouse is warming up` screen.

OS: macOS Mojave v10.14.6
Chrome version: Version 81.0.4044.9 (Official Build) dev (64-bit)

Is there anyone working on this thing?

**Comment by digiltd** [comment]
I also added `chrome-devtools-frontend.appspot.com` to my whitelist in pihole which fixed it for me.

Though I was only getting stuck on the "warming up" message on certain sites. Typically Magento 2, Lighthouse audits were working on other sites, just not the ones I was working on :(

**Comment by genehack** [comment]
Whitelisting `chrome-devtools-frontend.appspot.com` in my Pi-hole also fixed this problem for me. 

The only thing I don't understand is why the CLI version of lighthouse would continue to work -- it must not depend on having that domain resolve? 

(Thanks to everybody that continued to chip away at this!)

**Comment by connorjclark** [comment]
`chrome-devtools-frontend.appspot.com` is a service used by DevTools to serve "remove modules". These are files that are relatively large, so they are omitted from the default Chrome installation and only downloaded if DevTools is opened & requires it.

> The only thing I don't understand is why the CLI version of lighthouse would continue to work -- it must not depend on having that domain resolve?

Correct, the CLI has no need to defer code. In this case, Lighthouse is bundled and served to DevTools as a remote module, so it gets downloaded lazily.

It's unfortunate that this endpoint found its way into an adblocker. AFAIU, there is no analytics on this domain. It serves things like Lighthouse, and mobile device frames, and older versions of DevTools (for when you connect to a mobile device).

**Comment by holysoles** [comment]
Thanks for the help @xPaw that worked perfectly. I should have checked my pihole first but its a lesson learned!

**Comment by xPaw** [comment]
> It's unfortunate that this endpoint found its way into an adblocker.

It was removed from the adaway list - https://github.com/AdAway/adaway.github.io/issues/46

Devtools should be changed to display fetch errors when it's unable to fetch the code from that domain, so it doesn't just show "lighthouse is warming up".

**Comment by jj-wd** [comment]
While I'm happy that some of you found one root cause for this, I just wanted to mention that I'm not using any AdBlockers on any of the affected browsers or in my network and the issue is still appearing. It's not the biggest deal though, as one of the three Chrome based browsers I have installed (latest Chrome, Canary and Edge Beta) is always working while another randomly fails in doing an audit at the same time.

**Comment by 0xRaf** [comment]
Same issue here.
Chrome 80.0.3987.106,
Windows 10.0.18363.657.

Deleting all navigation data (Ctrl + Shift + Del) doesn't change anything.
Starting Chrome as an Administrator doesn't change anything.
Starting Lighthouse on google.com or a local basic index.html file either.
Disconnecting my VPN doesn't change anything.
There is even nothing unusual in the Task Manager (Chrome uses < 2 % CPU and < 300 MB RAM)

I tried to do a complete fresh install of Chrome :
- Uninstalled it
- Did a Disk Cleanup as an Administrator and checked everything
- Did a CCleaner as an Administrator and checked everything in files and registry
- Tried to find Chrome remnants in "This Computer" (found nothing)
- Rebooted
- Downloaded and installed the latest Chrome version (80.0.3987.106)
- Started it and went to google.com (no plugins installed, no configuration, nothing)
- Started Lighthouse and clicked on Generate report
- "Lighthouse is warming up" forever

I have no clue.

**Comment by davinderpalrehal** [comment]
Thanks @0xRaf , i disabled my VPN and my Adblocker and Lighthouse started working. It does suck that I need to disable VPN to actually get this to work.

**Comment by baash05** [comment]
I just updated chrome to  81.0.4044.113 (Official Build) (64-bit)
On Mac Book Pro

**Comment by 0xRaf** [comment]
@baash05 Yes, big news, I just got a Lighthouse report in Chrome to 81.0.4044.113 (Official Build) (64-bit), on Windows 10, with my VPN on, its "Anti-Tracker" on, and uBlock on !

**Comment by buhichan** [comment]
latest chrome, latest macos, no vpn, no ad-blocker, still get this issue. 
When lighthouse was not built in chrome devtool's audit panel, it worked so perfectly. 
Is it because lighthouse is trying to send data to google server while I'm living behind China Great Fire Wall? If so, is is possible to have an option of "offline mode"?

**Comment by patrickhulce** [comment]
> Is it because lighthouse is trying to send data to google server while I'm living behind China Great Fire Wall?

Not sending data, but Lighthouse is not bundled with Chrome and needs to be downloaded. This message appears while Lighthouse is being downloaded, so if something is blocking the request like the great fire wall then that could certainly explain it.

**Comment by avimehenwal** [comment]
I got the same problem on Brave running on Ubuntu Linux focal, **but seems to work fine on official google-chrome build 81.0**. :earth_americas: 

Might be an issue unrelated to chrome, still posting it here as 'General Information' as I see a couple of people mentioning Brave browser issue here. Just to confirm that lighthouse works for me on official google-chrome build 81 as of 29th april 2020 :date: 

**Version Details**
```
> brave-browser --version
Brave Browser 81.1.7.98 

> lsb_release --all
No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 20.04 LTS
Release:	20.04
Codename:	focal

> google-chrome --version
Google Chrome 81.0.4044.129 
```

**Comment by bsclifton** [comment]
@avimehenwal to get this working on Brave, you'll have to enable the `Remote Debugging` option under brave://settings/privacy

**Comment by avimehenwal** [comment]
@bsclifton Excellent! That works for me :+1: Thankyou for the solution

**Comment by xialvjun** [comment]
> > Is it because lighthouse is trying to send data to google server while I'm living behind China Great Fire Wall?
> 
> Not sending data, but Lighthouse is not bundled with Chrome and needs to be downloaded. This message appears while Lighthouse is being downloaded, so if something is blocking the request like the great fire wall then that could certainly explain it.

But I have downloaded Lighthouse(I mean the audit tab rather than the extension). I have audit a page with VPN connected. Then I want to audit another page with VPN disconnected, it still shows "lighthouse is warming up" forever.

**Comment by patrickhulce** [comment]
Thanks for the details @xialvjun I can reproduce this behavior in Chrome Canary as well. We're looking into it.

**Comment by lew-kni** [comment]
@patrickhulce - is there any update on this?
Getting the 'Lighthouse is warming up' issue on mac for the versions:
Version 83.0.4103.61 (Official Build) (64-bit)
Version 84.0.4147.30 (Official Build) beta (64-bit)

I'm testing from behind the great firewall which I've read on this thread that this may be an issue.

**Comment by patrickhulce** [comment]
I filed a Chromium bug for it https://bugs.chromium.org/p/chromium/issues/detail?id=1082934

I'm personally running into this issue more frequently even without the great firewall blocking me, so I hope we may tackle it soon.

**Comment by shawnlauzon** [comment]
I was having this problem, but for me using Canary in Incognito mode worked, while Google Chrome in any mode and Canary in normal mode caused the forever Warming Up problem 

Version 85.0.4166.0 (Official Build) canary (64-bit)

**Comment by fcFn** [comment]
Lighthouse works in Edge and at the same time doesn't work in Chrome (though that only happens occasionally). Weird.

**Comment by mariorodriguespt** [comment]
Having the same issue on Ubuntu with the lastest chrome version.

**Comment by Matt-Deacalion** [comment]
Same problem on Arch Linux with Chromium and Chrome. No adblocking or pi-hole.

**Versions**:

`Chromium 84.0.4147.135`
`Google Chrome 80.0.3987.132`

**uname -rv**:

`5.8.1-arch1-1 #1 SMP PREEMPT Wed, 12 Aug 2020 18:50:43 +0000`

**Comment by fengerzh** [comment]
Chrome 85 still have same problem. Mac OSX High Sierra, disable adblock, still cannot use, please.

**Comment by TravelingTechGuy** [comment]
👍 on adding `chrome-devtools-frontend.appspot.com` to PiHole.

Brave Version 1.13.82 Chromium: 85.0.4183.83 (Official Build) (64-bit)

**Comment by Mavrick2020** [comment]
Doesn't work for me. 
Chrome: Google Chrome is up to date
Version 85.0.4183.83 (Official Build) (32-bit)
OS: Windows 10
Lighthouse:100.0.0.0

**Comment by TravelingTechGuy** [comment]
@Mavrick2020 I think the root cause is (usually) tht this domain is unreachable by the browser you're running Lighthouse in.

Could you try opening a console window, and pinging that address? If you get a reply, there may be a different cause.

**Comment by Mavrick2020** [comment]
@TravelingTechGuy the domain is reachable, I can browse desired pages. The ping is successful as well.

**Comment by TravelingTechGuy** [comment]
@Mavrick2020 I have nothing smart to add then. In my case, the domain was blocked. The tool definitely makes use of external resources. It could be that other domains are blocked. I suggest re-opening the issue once you verified nothing gets blocked by your FW/VPN/PiHole/AdBlocker etc.

**Comment by mariorodriguespt** [comment]
PiHole was blocking the domain `chrome-devtools-frontend.appspot.com`,  now it is working.

**Comment by Mavrick2020** [comment]
Issue still persist at my end.
@TravelingTechGuy I am using Windows 10. How do I check for blocker? Also, is there any log created?

**Comment by lalosh** [comment]
same issue on 

chrome: Version 85.0.4183.102 (Official Build) (64-bit)
OS: kubuntu

updating chrome and disabling all extension and even trying chromium didn't work for me.

did anyone find  a solution for this issue?

**Comment by connorjclark** [comment]
Should we put a note about PiHole/firewall/w.e. after x seconds of this message?

**Comment by patrickhulce** [comment]
Yeah an error timeout with a message about PiHole/firewall sounds great @connorjclark  👍

**Comment by lmdcao** [comment]
CloudReady / Chromium OS, with Chromium browser. Same issue.

**Comment by Simonwep** [comment]
Same issue here with the `dev` version....
* Google Chrome: `87.0.4259.3 (Official Build) dev (64-bit) (cohort: Dev)`
* Revision: `dfeb7bdcaf32bc8e7efc0b690677db80d6fc556f-refs/branch-heads/4259@{#6}`
* OS: `Windows 10 OS Version 2004 (Build 19041.450)`
* JavaScript: `V8 8.7.75`

... and chrome stable:
* Google Chrome: `85.0.4183.83 (Official Build) (64-bit) (cohort: Stable)`
* Revision: `94abc2237ae0c9a4cb5f035431c8adfb94324633-refs/branch-heads/4183@{#1658}`
* OS: `Windows 10 OS Version 2004 (Build 19041.450)`
* JavaScript: `V8 8.5.210.20`

**Comment by connorjclark** [comment]
@Simonwep are you using any proxies like PiHole?

**Comment by Simonwep** [comment]
@connorjclark Oh man, yeah I should have checked that - when disabling it it works. The error I get when I run it and PiHole is active is the followint:

![image](https://user-images.githubusercontent.com/30767528/92972480-a33f9380-f482-11ea-9094-7300e0a88099.png)

 (Sorry the only time it crashed I made a screenshot, now it's working as the script is now probably cached).
Why would pihole be an issue in this case?

**Comment by connorjclark** [comment]
No worries, thanks for verifying!

cc @adamraine looks like we do get an explicit error when a proxy blocks it. Not sure how to catch this from the Lighthouse status panel, though.

**Comment by Lofesa** [comment]
Same here.
With Chrome Versión 87.0.4264.0 (Build oficial) canary (64 bits) it is warming for ever
With Chrome Versión 85.0.4183.102 (Build oficial) (64 bits) it works.
With Edge Versión 85.0.564.51 (Compilación oficial) (64 bits) it works.
No firewall involved as far as I have tried disabling it.
No matter if Chrome Canary is reinstalled.

**Comment by Simonwep** [comment]
The domain which gets blocked is `chrome-devtools-frontend.appspot.com` which is apparently part of [this blocklist](https://github.com/jerryn70/GoodbyeAds):
![image](https://user-images.githubusercontent.com/30767528/93483568-3c4a3080-f901-11ea-8dc7-0083e0208efc.png)

It works if you add `chrome-devtools-frontend.appspot.com` (exact domain is enough) to your whitelist.

**Comment by Lovor01** [comment]
It does not work for me with Chrome developer Version 87.0.4263.3 (Official Build) dev (64-bit).
no proxies, no add-blockers

**Comment by patrickhulce** [comment]
Lighthouse is currently broken in Canary, but the DevTools team is working on a fix.

**Comment by adamraine** [comment]
Lighthouse is no longer a remote module in m87:
https://chromiumdash.appspot.com/commit/419c91eff66647a4a6a16c3b5f558535c5568791

**Comment by Alaa84kh** [comment]
Disable these two options from chrome settings:

 - Continue running background apps when Google Chrome is closed 
 - Use hardware acceleration when available

Relaunch chrome then try again

That works for me

**Comment by Mavrick2020** [comment]
> Disable these two options from chrome settings:
> 
> * Continue running background apps when Google Chrome is closed
> * Use hardware acceleration when available
> 
> Relaunch chrome then try again
> 
> That works for me

I did changes in the settings as mentioned. However, it didn't help. Still facing the same issue.

**Comment by lmdcao** [comment]
I coulc make it work with the Chrome extension https://chrome.google.com/webstore/detail/lighthouse/blipmdconlkpinefehnmjammfjpmpbjk

**Comment by patrickhulce** [comment]
As https://github.com/GoogleChrome/lighthouse/issues/9484#issuecomment-713830083 states, Lighthouse shouldn't show this message moving forward in *Chrome* once m87 hits stable (~early December 2020). In the meantime, you can use Chrome Canary if this is a limitation for your environment.

**Comment by omidazm94** [comment]
> Disable these two options from chrome settings:
> 
> * Continue running background apps when Google Chrome is closed
> * Use hardware acceleration when available
> 
> Relaunch chrome then try again
> 
> That works for me

that worked for me. thanks ((:

**Comment by ibraindevs** [comment]
We faced the same issue. We once ran the lighthouse for www.google.com, to check its functionality, and then we lighthoused our sites. It audited the site and generated the report. 

If you still face the same issue, start interacting with the site while lighthouse audits, this solves the issue.
Hope this was useful.

Thanks,
iBrain Developers.

**Comment by hibrido-ivan** [comment]
It's basically completely random for me, closing the tab and trying again until the process really start in 2 second frame window is my go on for now

**Comment by ivanaugustobd** [comment]
It's basically completely random for me, closing the tab and trying again until the process really start in 2 second frame window is my go on for now

**Comment by AdamZaczek** [comment]
I'm having this issue now, after updating chrome.

**Comment by xinthose** [comment]
It worked for me when I used an incognito tab on my website.

**Comment by adamraine** [comment]
For anyone encountering this recently, we have an issue open for it https://github.com/GoogleChrome/lighthouse/issues/13154

**Comment by hanh090** [comment]
I solve this problem by kill Chrome and restart again

**Comment by matthew-dean** [comment]
Getting this issue after not having it before on Chrome Version 112.0.5615.137 (Official Build) (arm64)

**Comment by adamraine** [comment]
Hey folks I opened a new issue for this happening in Chrome 112 https://github.com/GoogleChrome/lighthouse/issues/15026

I'm going to lock this thread because it is very old, if this pops up again please file a new issue.
